pyiron_base/project/generic.py

```diff
@@ -512,34 +512,53 @@ def inspect(self, job_specifier):
         """
         return self.load(job_specifier=job_specifier, convert_to_object=False)
 
-    def iter_jobs(self, path=None, recursive=True, convert_to_object=True, status=None, job_type=None, progress=True):
+    def get_filtered_job_ids(self, recursive: bool = True, **kwargs: dict) -> list:
```

Comment:
```suggestion
    def get_filtered_job_ids(self, recursive: bool = True, **kwargs: dict) -> List[int]:
```
`List` comes from `typing`.

Reply:
I'm a python type hinting n00b so thanks for this.